centos/common/install_mpis.sh

```diff
@@ -30,27 +30,15 @@ cd openmpi-${OMPI_VERSION}
 ./configure --prefix=${INSTALL_PREFIX}/openmpi-${OMPI_VERSION} --with-ucx=${UCX_PATH} --with-hcoll=${HCOLL_PATH} --enable-mpirun-prefix-by-default --with-platform=contrib/platform/mellanox/optimized && make -j$(nproc) && make install
 cd ..
 
-
-# Intel MPI 2019 (update 6)
-IMPI_2019_VERSION="2019.6.166"
-wget http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/16120/l_mpi_${IMPI_2019_VERSION}.tgz
+# Intel MPI 2019 (update 7)
+IMPI_2019_VERSION="2019.7.217"
+wget http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/16546/l_mpi_${IMPI_2019_VERSION}.tgz
 tar -xvf l_mpi_${IMPI_2019_VERSION}.tgz
 cd l_mpi_${IMPI_2019_VERSION}
 sed -i -e 's/ACCEPT_EULA=decline/ACCEPT_EULA=accept/g' silent.cfg
 ./install.sh --silent ./silent.cfg
 cd ..
 
-
```

Comment:
Let's not remove this for the CentOS 7.* images.

Reply:
It is not removed, it is only moved to /azhpc-images/centos/centos-7.x/common/install_mpis.sh